bridgesync/bridgesync.go

```diff
@@ -55,6 +55,8 @@ type BridgeSync struct {
 	originNetwork uint32
 	reorgDetector ReorgDetector
 	blockFinality etherman.BlockNumberFinality
+	ethClient     aggkittypes.EthClienter
```

Comment:
In case we capture the bridge SC binding, we don't need this field as well?
```suggestion
        bridge *polygonzkevmbridgev2.Polygonzkevmbridgev2
```